lib/covalence/environment_tasks.rb

```diff
@@ -32,7 +32,7 @@ def run
 
       environments.each do |environment|
         # We do not want to render individual tasks for the reserved 'ci' and 'spec' namespaces, or any other specified with COVALENCE_RESERVED_NAMESPACE
-        break if task.has_key? 'environment' && RESERVED_NS.include?(task['environment'])
+        break if RESERVED_NS.include?(task['environment'])
```

Comment:
Will this break tasks that have 'environment' in the name?

Reply:
mmm, don't think that was what was removed, the removed piece is probably the equivalent to `task['environment'] != nil`, not `task['environment'] == 'environment'`. TBH I'm not quite sure why that check was there in the first place, and removing it didn't break any of the tests (terrible excuse, i know). I looked further down and it seems like if there were problems, it'd probably throw an exception down the line and it'd most likely be due to malformed hiera hierarchy setup.